core/src/main/java/nl/nn/adapterframework/pipes/AbstractPipe.java

```diff
@@ -254,6 +226,8 @@ public PipeForward findForward(String forward){
 		if (pipeline==null) {
 			return null;
 		}
+
+		//Remove global 'ForceFixedForwarding' forwards. 
```

Comment:
Dit comment is hier niet relevant

Reply:
Ik vind het op zich wel handig iets te hebben staan waarom dit gebeurd.